README.md

```diff
@@ -8,7 +8,21 @@ Implement landing page according to [Figma design](https://www.figma.com/file/hh
 - Tablet 1024
 - Mobile (> 320px)
 
-## Instructions
+
+## HR important moments
+- Скорость анимаций на всем лендинге одинаковая (например увеличение при наведении или выезд блоков при скроле)
```

Comment:
eng plz. we are preparing english version of the course and all tasks have to be in english